src/server.ts

```diff
@@ -53,6 +58,30 @@ async function getUser(userId: GetUserInput): Promise<User> {
   return user;
 }
 
+async function getUsers(usersInput: UserListInput): Promise<UserList> {
+  const limit = usersInput?.userLimit ? usersInput.userLimit : 10;
+  const offset = usersInput?.offset ? usersInput.offset : 0;
```

Comment:
```suggestion
  const offset = usersInput?.offset ?? 0;
```

Mesmo operador: nullish coalescing.